.travis.yml

```diff
@@ -19,4 +19,6 @@ script:
   - ./bin/phpunit --configuration tests/phpunit.xml.dist
   - ./bin/sabre-cs-fixer fix . --dry-run --diff
 
-before_script: composer install
+before_script:
+  - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
```

Comment:
we cannot use the proper travis `phpenv` api, because it doesnt work with hhvm right now